src/lib/components/TestTimer.svelte

```
@@ -1,34 +1,150 @@
+				body: JSON.stringify({ candidate, event })
+			});
+
+			if (!response.ok) return;
+
+			const data = await response.json();
+			if (typeof data.time_left !== 'number') return;
+			if (requestId < latestAppliedSyncRequestId) return;
+
+			latestAppliedSyncRequestId = requestId;
+			updateTimeLeft(Math.min(timeLeft, data.time_left));
+		} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)